contracts/test/MapleToken.t.sol

```diff
@@ -26,33 +20,28 @@ contract MapleTokenUser {
 
 }
 
-contract MapleTokenTest is DSTest {
+contract MapleTokenTest is MapleTest {
 
-    Hevm hevm;
     MapleToken token;
     MapleTokenUser usr;
 
-    uint256 constant WAD = 10 ** 18;
+    uint256 holder1Pk = 1;
+    uint256 holder2Pk = 2;
+    uint256 nonce     = 0;
+    uint256 deadline  = 5000000000; // timestamp far in the future
```

Comment:
Two spaces between ; and // and capital first letter:
`uint256 deadline = 5000000000;  // Timestamp far in the future`